src/startup_api.py

```
@@ -0,0 +1,338 @@
+            "joy", "love", "nervousness", "optimism", "pride", "realization",
+            "relief", "remorse", "sadness", "surprise", "neutral"
+        ]
+
+        emotion_scores = predictions[0].tolist()
+        return {
+            "text": text,
+            "emotions": dict(zip(emotion_labels, emotion_scores)),
+            "predicted_emotion": emotion_labels[emotion_scores.index(max(emotion_scores))]
+        }
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)